sys/build.sh

```diff
@@ -59,24 +59,18 @@ A=$(dirname "$PWD/$0")
 cd "$A" && cd .. || exit 1
 
 if [ "`uname`" = Darwin ]; then
-	DEFAULT_PREFIX=/usr/local
 	# purge previous installations on other common paths
 	if [ -f /usr/bin/r2 ]; then
 		type sudo || NOSUDO=1
 		[ "$(id -u)" = 0 ] || SUDO=sudo
 		[ -n "${NOSUDO}" ] && SUDO=
 		# purge first
 		echo "Purging r2 installation..."
-		./configure --prefix=/usr > /dev/null
+		./configure > /dev/null
 		${SUDO} ${MAKE} uninstall > /dev/null
 	fi
-else
-	DEFAULT_PREFIX=/usr
-	[ -n "${PREFIX}" -a "${PREFIX}" != /usr ] && \
-		CFGARG="${CFGARG} --with-rpath"
```

Comment:
this will break the behaviour in many linux distros

Reply:
the rpath is necessary only when building outside the default path.. this is, if u install in /usr/local, you NEED rpath, so this change is wrong